resources/i18n/hi_HI/fdmprinter.def.json.po

```diff
@@ -0,0 +1,8786 @@
+
+#: fdmprinter.def.json
+msgctxt "machine_heat_zone_length label"
+msgid "Heat Zone Length"
+msgstr "हीट जोन की लंबाई"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_heat_zone_length description"
+msgid "The distance from the tip of the nozzle in which heat from the nozzle is transferred to the filament."
+msgstr "नोजल की नोक से दूरी जो फिलामेंट में हीट को स्थानांतरित करती है।"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_nozzle_temp_enabled label"
+msgid "Enable Nozzle Temperature Control"
+msgstr "नोजल तापमान नियंत्रण सक्षम करें"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_nozzle_temp_enabled description"
+msgid "Whether to control temperature from Cura. Turn this off to control nozzle temperature from outside of Cura."
+msgstr "Cura से तापमान नियंत्रित करना है या नहीं। Cura के बाहर से नोजल के तापमान को नियंत्रित करने के लिए इसे बंद कर दें।"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_nozzle_heat_up_speed label"
+msgid "Heat Up Speed"
+msgstr "हीट अप स्पीड"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_nozzle_heat_up_speed description"
+msgid "The speed (°C/s) by which the nozzle heats up averaged over the window of normal printing temperatures and the standby temperature."
+msgstr "वह गति (डिग्री सेल्सियस/सेक) जिससे नोज़ल गर्म होता है, सामान्य मुद्रण तापमान और स्टैंडबाय तापमान की खिड़की पर औसत होता है।"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_nozzle_cool_down_speed label"
+msgid "Cool Down Speed"
+msgstr "कूल डाउन स्पीड"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_nozzle_cool_down_speed description"
+msgid "The speed (°C/s) by which the nozzle cools down averaged over the window of normal printing temperatures and the standby temperature."
+msgstr "गति (°C/s) जिसके द्वारा नोज़ल ठंडा हो जाता है, सामान्य मुद्रण तापमान और स्टैंडबाय तापमान की खिड़की पर औसत होता है।"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_min_cool_heat_time_window label"
+msgid "Minimal Time Standby Temperature"
+msgstr "स्टैंडबाय मोड में न्यूनतम तापमान"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_min_cool_heat_time_window description"
+msgid "The minimal time an extruder has to be inactive before the nozzle is cooled. Only when an extruder is not used for longer than this time will it be allowed to cool down to the standby temperature."
+msgstr "नोज़ल के ठंडा होने से पहले एक्सट्रूडर को कम से कम समय तक निष्क्रिय रहना चाहिए। इसे स्टैंडबाय तापमान तक ठंडा करने के लिए, एक्सट्रूडर को सेट से अधिक समय तक निष्क्रिय रहना चाहिए।"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_gcode_flavor label"
+msgid "G-code Flavor"
+msgstr "G-कोड प्रकार"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_gcode_flavor description"
+msgid "The type of g-code to be generated."
+msgstr "जी-कोड का प्रकार जनरेट किया जाना है।"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_gcode_flavor option RepRap (Marlin/Sprinter)"
+msgid "Marlin"
+msgstr "मार्लिन"
+
+
+#: fdmprinter.def.json
+msgctxt "machine_gcode_flavor option RepRap (Volumetric)"
+msgid "Marlin (Volumetric)"
+msgstr "मार्लिन (Volumetric)"
```

Comment:
Marlin is a firmware name and should not be translated.
But Volumetric might be translated?